src/current/_includes/releases/v23.2/v23.2.13.md

```diff
@@ -16,7 +16,7 @@ Release Date: October 17, 2024
 
       [#130664][#130664]
 
-- The new [metric]({% link v23.2/metrics.md %}) `changefeed.total_ranges` allows observation of the number of changes that are watched by a changefeed aggregator. It uses the same polling interval as `changefeed.lagging_ranges`, which is controlled by the changefeed option `lagging_ranges_polling_interval`. [#130984][#130984]
+- The new [metric]({% link v23.2/metrics.md %}) `changefeed.total_ranges` allows observation of the number of ranges that are watched by a changefeed aggregator. It uses the same polling interval as `changefeed.lagging_ranges`, which is controlled by the changefeed option `lagging_ranges_polling_interval`. [#130984][#130984]
```

Comment:
Thanks for catching that Andy, I updated that and checked the other release note version and they're all correct 👍